doc/productivity.md

```diff
@@ -176,7 +176,9 @@ When looking at other's pull requests, it may make sense to add the following se
         url = git@github.com:bitcoin/bitcoin.git
 ```
 
-This will add an `upstream-pull` remote to your git repository, which can be fetched using `git fetch --all` or `git fetch upstream-pull`. Afterwards, you can use `upstream-pull/NUMBER/head` in arguments to `git show`, `git checkout` and anywhere a commit id would be acceptable to see the changes from pull request NUMBER.
+This will add an `upstream-pull` remote to your git repository, which can be fetched using `git fetch --all` or `git fetch upstream-pull`. It will download ~800 MB (all PRs, including merged and closed ones) and will expand the disk usage from ~180 MB to ~1400 MB. Afterwards, you can use `upstream-pull/NUMBER/head` in arguments to `git show`, `git checkout` and anywhere a commit id would be acceptable to see the changes from pull request NUMBER.
```

Comment:
Maybe it's better to not specify exact numbers and use something like "hundreds of megabytes" instead, as it will change in future?

Reply:
Done. Shortened to just "It will download and store on disk about 1 GB of data" - enough to avoid the surprise.